projects/aca-content/src/lib/components/dl-custom-components/name-column-badges/name-column-badges.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="aca-name-column-badges-container">
```

Comment:
Since we're moving this component as shared one we should also change the name of this class as it no longer will be located only at name column